src/components/Headline/Headline.module.scss

```diff
@@ -11,6 +11,7 @@
   text-align: center;
 
   h1 {
+    font-family: var(--ifm-heading-font-family);
```

Comment:
Only `h1` tag should have Apax? how about `h2`, `h3` etc?

Reply:
This is only for the main page. The other h3 elements have the Apax font applied ;)
<img width="706" alt="image" src="https://user-images.githubusercontent.com/477945/203738625-905c79c8-1779-4932-b55f-4553f2e59097.png">
<img width="1099" alt="image" src="https://user-images.githubusercontent.com/477945/203738852-4872001c-bf58-4163-b0d2-2acfb149a048.png">